pinot-core/src/main/java/com/linkedin/pinot/core/operator/filter/StarTreeIndexOperator.java

```diff
@@ -343,6 +343,11 @@ public void setEndDocId(int endDocId) {
           }
 
           @Override
+          public long getNumEntriesScannedInFilter() {
```

Comment:
For now, there is no support for StarTree and Realtime use case. I will try to add them into this PR.